python/lsst/pipe/tasks/quickFrameMeasurement.py

```diff
@@ -227,6 +237,42 @@ def _calcMedianXxYy(objData):
         medianYy = np.nanmedian([element['xx'] for element in objData.values()])
         return medianXx, medianYy
 
+    @staticmethod
+    def _getCentreOfMass(exp, nominalCentroid, boxSize):
+        """Get the centre of mass around a point in the image.
+
+        Parameters
+        ----------
+        exp : `lsst.afw.image.Exposure`
+            The exposure in question.
+
+        nominalCentroid : `tuple` of `float`
+            Nominal location of the centroid in pixel coordinates.
+
+        boxSize : `int`
+            The size of the box around the nominalCentroid in which to measure
+            the centre of mass.
+
+        Returns
+        -------
+        com : `tuple` of `float`
+            The locaiton of the centre of mass of the brightest source in pixel
+            coordinates.
+        """
+        centroidPoint = geom.Point2I(nominalCentroid)
+        extent = geom.Extent2I(1, 1)
+        bbox = geom.Box2I(centroidPoint, extent)
+        bbox = bbox.dilatedBy(int(boxSize//2))
```

Comment:
Isn't `boxsize//2` already going to be an `int`? I think this means the casting you are doing here is because you can't trust the caller to give you an int despite the parameter typing. Might be best to change the docstring to say that it accepts int or float.

Reply:
The type of a (float//2) is still a float, albeit one with a decimal part equal to zero, so the `int` cast for the `dilatedBy()` call which only accepts true ints is still sadly necessary.
I can change the doc too though, as it really doesn't matter what's passed (it gets multiplied by 1.3 anyway so likely gets floatified anyhow)

Reply:
Yes, my point was that the int exists because you don't trust the caller to give you an int and therefore you may as well give up and admit that you routine accepts any number type.

Reply:
Well, that's not primarily why that's there - even if I get an int, I multiply it by 1.3, so it's no longer an int at that point, so it always need that call regardless. But sure, I'll make the doc read int or float too, because it really doesn't matter.

Reply:
Now that I look, you should have simply said that parameter was a float and not an int and everything would be fine. It was incorrectly documenting the parameter as an int that started this whole discussion.

Reply:
Are you saying I should go and change the type in the pexConfig at the top too? Why does any of this matter at all?

Reply:
I'm a tad confused. All I wanted was the docstring to say it works for int and float -- you were already calling this routine with a float and not an int. pex_config will validate the type you want it to be and the context for that config item doesn't seem to have any bearing on the call signature of this function.

Reply:
OK, it just seemed like, given that they're used in the same place in the same way (depending on which is supplied), that they should match. Honestly though, none of this actually matters, it's not even that likely that anyone will ever even change or supply this parameter in reality (let alone have its type matter), it's just not OK to hard code things.

Reply:
I'll leave it as an int in the config, and an "int of float" in the `run()` docstring for now, unless you think something else needs changing.